clang/test/CodeGen/cheri/hybrid-cap-varargs.c

```diff
@@ -0,0 +1,586 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
```

Comment:
Since this change doesn't really affect any existing tests, it might be good to add this test in a previous commit so that the fix only shows the diff? (Generating the MIPS code with `-Wno-error=mips-cheri-bugs`)

This seems to be the common approach upstream for autogenerated tests.